source/ada/lsp-ada_handlers.adb

```diff
@@ -792,8 +792,8 @@ package body LSP.Ada_Handlers is
          Self.Show_Message
            (VSS.Strings.To_Virtual_String
               ("More than one .gpr found." & Line_Feed &
-                 "Note: you can configure a project " &
-                 " through the ada.projectFile setting."));
+               "Configure using the ada.chooseProject command" & Line_Feed &
+               " or the ada.projectFile setting."));
```

Comment:
On second thought, we probably don't want to mention vscode specific stuff here. `ada.projectFile` is actually what you need to set in all LSP clients.